docs/api/input.md

```diff
@@ -134,6 +134,14 @@ import Migration from '@site/static/usage/v7/input/migration/index.md';
 
 Ionic uses heuristics to detect if an app is using the modern input syntax. In some instances, it may be preferable to continue using the legacy syntax. Developers can set the `legacy` property on `ion-input` to `true` to force that instance of the input to use the legacy syntax.
 
+## Input Masking
```

Comment:
Can we move Input Masking up above Theming?

Reply:
+1 This is a big feature so we should prioritize the content